src/test/regress/expected/multi_transaction_recovery_multiple_databases.out

```diff
@@ -0,0 +1,360 @@
+DETAIL:  Citus does not propagate CREATE DATABASE command to workers
+HINT:  You can manually create a database and its extensions on workers.
+SELECT oid AS db2_oid
+FROM pg_database
+WHERE datname = 'db2'
+\gset
+\c - - - :worker_1_port
+CREATE DATABASE db2;
+NOTICE:  Citus partially supports CREATE DATABASE for distributed databases
+DETAIL:  Citus does not propagate CREATE DATABASE command to workers
+HINT:  You can manually create a database and its extensions on workers.
+\c - - - :worker_2_port
+CREATE DATABASE db2;
+NOTICE:  Citus partially supports CREATE DATABASE for distributed databases
+DETAIL:  Citus does not propagate CREATE DATABASE command to workers
+HINT:  You can manually create a database and its extensions on workers.
+\c db2 - - :worker_1_port
+CREATE EXTENSION citus;
+\c db2 - - :worker_2_port
+CREATE EXTENSION citus;
+\c db2 - - :master_port
+CREATE EXTENSION citus;
+SELECT citus_add_node('localhost', :worker_1_port);
+ citus_add_node
+---------------------------------------------------------------------
+              1
+(1 row)
+
+SELECT citus_add_node('localhost', :worker_2_port);
+ citus_add_node
+---------------------------------------------------------------------
+              2
+(1 row)
+
+SELECT current_database();
+ current_database
+---------------------------------------------------------------------
+ db2
+(1 row)
+
+SELECT pg_sleep_for('5 SECONDS');
+ pg_sleep_for
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT *
+FROM pg_dist_node;
+ nodeid | groupid | nodename  | nodeport | noderack | hasmetadata | isactive | noderole | nodecluster | metadatasynced | shouldhaveshards
+---------------------------------------------------------------------
+      1 |       1 | localhost |    57637 | default  | t           | t        | primary  | default     | t              | t
+      2 |       2 | localhost |    57638 | default  | t           | t        | primary  | default     | t              | t
+(2 rows)
+
+SELECT groupid AS worker_1_group_id
+FROM pg_dist_node
+WHERE nodeport = :worker_1_port;
+ worker_1_group_id
+---------------------------------------------------------------------
+                 1
+(1 row)
+
+\gset
+SELECT groupid AS worker_2_group_id
+FROM pg_dist_node
+WHERE nodeport = :worker_2_port;
+ worker_2_group_id
+---------------------------------------------------------------------
+                 2
+(1 row)
+
+\gset
+-- Prepare transactions on first database
+\c db1 - - :worker_1_port
+BEGIN;
+CREATE TABLE should_abort
+(
+    value int
+);
+SELECT 'citus_0_1234_0_0_' || :'db1_oid' AS transaction_1_worker_1_db_1_name
+\gset
+PREPARE TRANSACTION :'transaction_1_worker_1_db_1_name';
+BEGIN;
+CREATE TABLE should_commit
+(
+    value int
+);
+SELECT 'citus_0_1234_1_0_' || :'db1_oid' AS transaction_2_worker_1_db_1_name
+\gset
+PREPARE TRANSACTION :'transaction_2_worker_1_db_1_name';
+BEGIN;
+CREATE TABLE should_be_sorted_into_middle
```

Comment:
the original test file could have used way more comments (my bad), but the name "should_be_sorted_into_middle" was mainly relevant as a prepared transaction name to cover edge cases around binary search in the transaction recovery logic (between abort and commit)

the table name was always irrelevant and in this context it seems quite confusing since the prepared transaction name below does not sort into the middle of the list

Reply:
Addressed: removed those as not relevant to the test case